lib/taste_tester/ssh.rb

```diff
@@ -68,7 +67,7 @@ def cmd
       end
       cmds = @cmds.join(' && ')
       cmd = "#{TasteTester::Config.ssh_command} " +
-            "-T -o BatchMode=yes -o ConnectTimeout=#{@timeout} " +
+            '-T -o BatchMode=yes ' +
```

Comment:
we should still pass in a connecttimeout, even if its the one from the config?

Reply:
You can override this things by changing your ssh_config (either the one in `/etc/ssh` or the one in `~/.ssh`), or replace the SSH command with something that injects this parameter.

There is no default in the ssh_config man page, so I assume this will just the OS default TCP connect timeout?

I think we shouldn't specify this at all. Up to you.

Reply:
It's really really long... we set this to 5 seconds by default, IIRC, becasue otherwise when you tt on a bunch of hosts, if one is down, it waits for that one forever.... honestly, I've considered changing it to 2...

Reply:
I resolve this conversation.